be/src/common/config.h

```diff
@@ -895,4 +895,8 @@ CONF_Int32(exception_stack_level, "1");
 CONF_String(exception_stack_white_list, "std::");
 CONF_String(exception_stack_black_list, "apache::thrift::,ue2::,arangodb::");
 
+CONF_String(rocksdb_meta_cf_opts_string,
```

Comment:
Add a clear message about the problem, and solution, and result in the commit message